test/greet-test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const expect = require('chai').expect;
+const greet = require('../lib/greet.js');
+
+describe('testing greet module', function(){
+  describe('valid inputs', function() {
+    it('should return "Hello gnoevil"', function(){
+      expect(greet('gnoevil')).to.equal('Hello gnoevil');
```

Comment:
This test failed on me because you should be expecting ``Hello, gnoevil`` instead of ``Hello gnoevil.``